cob_moveit_config/bin/add_robot.sh

```diff
@@ -16,5 +16,5 @@ moveit_setup_assistant_config:
     package: cob_hardware_config
     relative_path: robots/ROBOT/urdf/ROBOT.urdf.xacro
   SRDF:
-    relative_path: moveit/config/ROBOT.srdf
```

Comment:
I think that it is not correct. The path is relative to the `.setup_assistant` file and this will be created under `robots/$robot/moveit/.setup_assistant` , being the relative path between the `.setup_assistant` and the srdf model `config/ROBOT.srdf` .

You can also change the path of the setup_assistant (line 13) or maybe use the following config to find the SRDF file:

```
   SRDF:
     package: cob_moveit_config
     relative_path: moveit/config/ROBOT.srdf
```